pkg/domain/domain.go

```diff
@@ -2801,13 +2822,14 @@ const (
 
 // NotifyUpdatePrivilege updates privilege key in etcd, TiDB client that watches
 // the key will get notification.
-func (do *Domain) NotifyUpdatePrivilege() error {
+func (do *Domain) NotifyUpdatePrivilege(userList []string) error {
 	// No matter skip-grant-table is configured or not, sending an etcd message is required.
 	// Because we need to tell other TiDB instances to update privilege data, say, we're changing the
 	// password using a special TiDB instance and want the new password to take effect.
 	if do.etcdClient != nil {
+		notifyList := strings.Join(userList, ", ")
```

Comment:
you use `,` above but here `, `. Maybe write a separate type for "user list" and its associated member functions for (un)marshal to find bugs easier 😂